tests/integration-tests/configs/common/common.yaml

```diff
@@ -186,12 +172,12 @@ createami:
     dimensions:
       - regions: ["eu-west-3"]
         instances: {{ common.INSTANCES_DEFAULT_X86 }}
-        oss: {{ common.OSS_ONE_PER_DISTRO }}
+        oss: ["ubuntu2004"]
   test_createami.py::test_kernel4_build_image_run_cluster:
     dimensions:
       - regions: ["eu-south-1"]
         instances: {{ common.INSTANCES_DEFAULT_X86 }}
-        schedulers: ["awsbatch", "slurm"]
+        schedulers: ["slurm"]
         oss: ["alinux2"]
   test_createami.py::test_build_image_custom_components:
```

Comment:
This test can be run only with 1 architecture an the logic to tests the component can be combined in one region.
* https://github.com/aws/aws-parallelcluster/blob/develop/tests/integration-tests/tests/createami/test_createami/test_build_image_custom_components/image.config.yaml#L7
* https://github.com/aws/aws-parallelcluster/blob/develop/tests/integration-tests/tests/createami/test_createami.py#L446

Reply:
Done in https://github.com/aws/aws-parallelcluster/pull/5201